database/schema.sql

```diff
@@ -0,0 +1,99 @@
+CREATE TABLE items(
+    id SERIAL PRIMARY KEY,
+    archived BOOLEAN DEFAULT 'f',
+    author_id SERIAL NOT NULL,
+    label_id SERIAL NOT NULL,
+    genre_id SERIAL NOT NULL,
+    publish_date TIMESTAMP NOT NULL,
+    CONSTRAINT author_fk FOREIGN KEY (author_id) REFERENCES author(id),
+    CONSTRAINT label_fk FOREIGN KEY (label_id) REFERENCES label(id),
+    CONSTRAINT genre_fk FOREIGN KEY (genre_id) REFERENCES genre(id)
+);
+
+
+CREATE TABLE genre(
+    id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(100),
+    item_id SERIAL NOT NULL,
+    CONSTRAINT item_fk FOREIGN KEY (item_id) REFERENCES items(id)
+);
+
+CREATE TABLE lables (
+    id INT PRIMARY KEY,
+    title VARCHAR(100) NOT NULL,
+    color VARCHAR(100) NOT NULL,
+);
+
+CREATE TABLE labels (
+    id  INT GENERATED ALWAYS AS IDENTITY,
+    title VARCHAR(100),
+    color VARCHAR(100),
+    PRIMARY KEY(id)
+);
+
+CREATE TABLE sources (
+    id INT PRIMARY KEY,
+    name VARCHAR(100) NOT NULL,
+);
+
+CREATE TABLE musicalbum (
+    id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(100),
+    publish_date DATE,
+    archived BOOLEAN,
+    on_spotify BOOLEAN DEFAULT 'f',
+    genre_id INT,
+    CONSTRAINT genre_fk FOREIGN KEY (genre_id) REFERENCES genre(id)
+);
```

Comment:
- [ ] According to the ERD, musicalbum has only the `id` and `on_spotify` columns. Kindly remove every other column here, please.

```suggestion
CREATE TABLE musicalbum (
    id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
    on_spotify BOOLEAN,
    FOREIGN KEY(item_id) REFERENCES items(id)
);
```